core/templates/pages/classroom-page/classroom-page.component.ts

```diff
@@ -100,7 +100,7 @@ export class ClassroomPageComponent implements OnDestroy {
           thumbnailFilename
         )
       : this.urlInterpolationService.getStaticImageUrl(
-          '/classrooms/default-classroom-thumbnail.svg'
+          '/classroom/default-classroom-thumbnail.svg'
```

Comment:
I did this as by mistake I created two separate folder classroom and classrooms.

Now we just have one single folder which has all the classroom related images.